### PR TITLE
Update exercise1.md

### PR DESCRIPTION
**Pull request by zhou-1**
In "Step 7: create a branch step", after create the branch, we should switch to the new created branch by using "git chechkout <branch_name>"; otherwise, we will add file on master.

### CODE REVIEW

File: 00-git/exercises/exercise1.md [hunk 1]
```diff
@@ -95,6 +95,10 @@ Verify the branch was created by listing the branches using the above command.
 
 <kbd> git branch </kbd> 
 
+Do not forget to switch to new branch you just created.    
+
+<kbd> git chechkout <branch_name> </kbd>
```
Comment (gallettilance) [inline 00-git/exercises/exercise1.md:100]: Looks like a little typo here: it should be `checkout`

File: 00-git/exercises/exercise1.md [hunk 2]
```diff
@@ -95,6 +95,10 @@ Verify the branch was created by listing the branches using the above command.
 
 <kbd> git branch </kbd> 
 
+Do not forget to switch to new branch you just created.
```
Comment (gallettilance) [inline 00-git/exercises/exercise1.md:98]:
```suggestion
Do not forget to switch to the new branch you just created. 
```

**Review by gallettilance — CHANGES_REQUESTED** [review]
Great work making this PR! I noticed a few typos. Please fix these when you have time and amend your commit

File: 00-git/exercises/exercise1.md (hunk 2 above)
Comment (zhou-1) [reply 00-git/exercises/exercise1.md:98]: I see, will change.

File: 00-git/exercises/exercise1.md (hunk 1 above)
Comment (zhou-1) [reply 00-git/exercises/exercise1.md]: Will change.

**Review by zhou-1 — COMMENTED** [review]
Modified typos.

**Review by gallettilance — APPROVED** [review]

**Pull request merged**